src/main/java/hu/bme/mit/spaceship/TorpedoStore.java

```diff
@@ -28,20 +28,21 @@ public TorpedoStore(int numberOfTorpedos){
     }
   }
 
+  Random generator = new Random();
```

Comment:
Should probably mark it explicitly private:
```suggestion
  private Random generator = new Random();
```